<<<<<<< HEAD
2024-06-18 13:37:44 database: wrong user or password; user: "SA"
org.h2.message.DbException: ユーザ名またはパスワードが不正です
Wrong user name or password [28000-200]
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.engine.Engine.openSession(Engine.java:131)
	at org.h2.engine.Engine.openSession(Engine.java:192)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:171)
	at org.h2.engine.Engine.createSession(Engine.java:166)
	at org.h2.engine.Engine.createSession(Engine.java:29)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:340)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:152)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:309)
	at org.h2.util.JdbcUtils.getConnection(JdbcUtils.java:289)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:785)
	at org.h2.server.web.WebApp.login(WebApp.java:1008)
	at org.h2.server.web.WebApp.process(WebApp.java:223)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: org.h2.jdbc.JdbcSQLInvalidAuthorizationSpecException: ユーザ名またはパスワードが不正です
Wrong user name or password [28000-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:461)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
<<<<<<< HEAD
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.mvstore.db.MVTable.addIndex(MVTable.java:334)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:151)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:06:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: 不明なデータ型: "BOOLEAM"
Unknown data type: "BOOLEAM"; SQL statement:
ALTER TABLE seas ADD COLUMN touban BOOLEAM [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:06:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "TENMEN" が重複しています
Duplicate column name "TENMEN"; SQL statement:
ALTER TABLE seas ADD COLUMN tenmen BOOLEAN [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:08:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: 不明なデータ型: 
Unknown data type: ; SQL statement:
ALTER TABLE seas ADD COLUMN herbBOOLEAN [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:33:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE COOKS_SEAS(
    COOKS_SEAS_NUM  INT(5)          NOT NULL,
    SEAS_NUM        INT(5)          NOT NULL,
    SOLTY           BOOLEAN         ,
    REGION          CHAR(10)
    FOREIGN[*] KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
)"; 期待されるステートメント "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE COOKS_SEAS(
    COOKS_SEAS_NUM  INT(5)          NOT NULL,
    SEAS_NUM        INT(5)          NOT NULL,
    SOLTY           BOOLEAN         ,
    REGION          CHAR(10)
    FOREIGN[*] KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
)"; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE cooks_seas(
    cooks_seas_num  INT(5)          not null,
    seas_num        INT(5)          not null,
    solty           BOOLEAN         ,
    region          CHAR(10)
    FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
) [42001-200]
2024-06-18 14:34:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "SEAS" が見つかりません
Table "SEAS" not found; SQL statement:
CREATE TABLE cooks_seas(
    cooks_seas_num  INT(5) NOT NULL,
    seas_num        INT(5) NOT NULL,
    solty           BOOLEAN,
    region          CHAR(10),
    FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
) [42102-200]
2024-06-18 14:49:27 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(user_num) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:129)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 15:15:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOURS_NUM INTEGER(5) NOT NULL,
ROUNDS_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOKS_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOURS_NUM,ROUNDS_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOURS_NUM) REFERENCES TOURS(TOURS_NUM)
FOREIGN KEY (ROUNDS_NUM) REFERENCES ROUNDS(ROUNDS_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOKS_NUM) REFERENCES COOKS(COOKS_NUM))"; 期待されるステートメント "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOURS_NUM INTEGER(5) NOT NULL,
ROUNDS_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOKS_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOURS_NUM,ROUNDS_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOURS_NUM) REFERENCES TOURS(TOURS_NUM)
FOREIGN KEY (ROUNDS_NUM) REFERENCES ROUNDS(ROUNDS_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOKS_NUM) REFERENCES COOKS(COOKS_NUM))"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tours_num INTEGER(5) NOT NULL,
rounds_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cooks_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tours_num,rounds_num,hole_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
FOREIGN KEY (user_num) REFERENCES users(user_num)
FOREIGN KEY (tours_num) REFERENCES tours(tours_num)
FOREIGN KEY (rounds_num) REFERENCES rounds(rounds_num)
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num)
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta)
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin)
FOREIGN KEY (cooks_num) REFERENCES cooks(cooks_num)) [42001-200]
2024-06-18 15:28:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE ROUNDS
(
ROUND_NUM INTEGER(5) NOT NULL AUTO_INCREMENT,
HOLE_NUM INTEGER(1) NOT NULL,
FOOD_SEAS_NUM INTEGER(5),
SEAS_NUM INTEGER(5),
COOK_NUM INTEGER(5),
SCORE_NUM INTEGER(5),
PLAY_STATUS VARCHAR(4) NOT NULL,
COURCE_NAME VARCHAR(10) ,
HOLE_DATE DATE(10)
FOREIGN[*] KEY (FOOD_SEAS_NUM) REFERENCES FOOD_SEAS(FOOD_SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM)
FOREIGN KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
)"; 期待されるステートメント "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE ROUNDS
(
ROUND_NUM INTEGER(5) NOT NULL AUTO_INCREMENT,
HOLE_NUM INTEGER(1) NOT NULL,
FOOD_SEAS_NUM INTEGER(5),
SEAS_NUM INTEGER(5),
COOK_NUM INTEGER(5),
SCORE_NUM INTEGER(5),
PLAY_STATUS VARCHAR(4) NOT NULL,
COURCE_NAME VARCHAR(10) ,
HOLE_DATE DATE(10)
FOREIGN[*] KEY (FOOD_SEAS_NUM) REFERENCES FOOD_SEAS(FOOD_SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM)
FOREIGN KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
)"; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE rounds
(
round_num INTEGER(5) NOT NULL AUTO_INCREMENT,
hole_num INTEGER(1) NOT NULL,
food_seas_num INTEGER(5),
seas_num INTEGER(5),
cook_num INTEGER(5),
score_num INTEGER(5),
play_status VARCHAR(4) NOT NULL,
cource_name VARCHAR(10) ,
hole_date DATE(10)
FOREIGN KEY (food_seas_num) REFERENCES food_seas(food_seas_num)
FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
) [42001-200]
2024-06-18 15:28:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "ROUNDS" が見つかりません
Table "ROUNDS" not found; SQL statement:
//複合主キーの設定
ALTER TABLE rounds ADD PRIMARY KEY(round_num , hole_num) [42102-200]
2024-06-18 15:29:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE ROUNDS
(
ROUND_NUM INTEGER(5) NOT NULL AUTO_INCREMENT,
HOLE_NUM INTEGER(1) NOT NULL,
FOOD_SEAS_NUM INTEGER(5),
SEAS_NUM INTEGER(5),
COOK_NUM INTEGER(5),
SCORE_NUM INTEGER(5),
PLAY_STATUS VARCHAR(4) NOT NULL,
COURCE_NAME VARCHAR(10) ,
HOLE_DATE DATE(10)
FOREIGN[*] KEY (FOOD_SEAS_NUM) REFERENCES FOOD_SEAS(FOOD_SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM)
FOREIGN KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
PRIMARY KEY(ROUNDS_NUM , HOLE_NUM))"; 期待されるステートメント "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE ROUNDS
(
ROUND_NUM INTEGER(5) NOT NULL AUTO_INCREMENT,
HOLE_NUM INTEGER(1) NOT NULL,
FOOD_SEAS_NUM INTEGER(5),
SEAS_NUM INTEGER(5),
COOK_NUM INTEGER(5),
SCORE_NUM INTEGER(5),
PLAY_STATUS VARCHAR(4) NOT NULL,
COURCE_NAME VARCHAR(10) ,
HOLE_DATE DATE(10)
FOREIGN[*] KEY (FOOD_SEAS_NUM) REFERENCES FOOD_SEAS(FOOD_SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM)
FOREIGN KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
PRIMARY KEY(ROUNDS_NUM , HOLE_NUM))"; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE rounds
(
round_num INTEGER(5) NOT NULL AUTO_INCREMENT,
hole_num INTEGER(1) NOT NULL,
food_seas_num INTEGER(5),
seas_num INTEGER(5),
cook_num INTEGER(5),
score_num INTEGER(5),
play_status VARCHAR(4) NOT NULL,
cource_name VARCHAR(10) ,
hole_date DATE(10)
FOREIGN KEY (food_seas_num) REFERENCES food_seas(food_seas_num)
FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
PRIMARY KEY(rounds_num , hole_num)) [42001-200]
2024-06-18 15:33:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE ROUNDS
(
ROUND_NUM INTEGER(5) NOT NULL AUTO_INCREMENT,
HOLE_NUM INTEGER(1) NOT NULL,
FOOD_SEAS_NUM INTEGER(5),
SEAS_NUM INTEGER(5),
COOK_NUM INTEGER(5),
SCORE_NUM INTEGER(5),
PLAY_STATUS VARCHAR(4) NOT NULL,
COURCE_NAME VARCHAR(10) ,
HOLE_DATE DATE(10),
FOREIGN KEY (FOOD_SEAS_NUM) REFERENCES FOOD_SEAS(FOOD_SEAS_NUM)
FOREIGN[*] KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM)
FOREIGN KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
PRIMARY KEY(ROUND_NUM , HOLE_NUM))"; 期待されるステートメント "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"
Syntax error in SQL statement "CREATE TABLE ROUNDS
(
ROUND_NUM INTEGER(5) NOT NULL AUTO_INCREMENT,
HOLE_NUM INTEGER(1) NOT NULL,
FOOD_SEAS_NUM INTEGER(5),
SEAS_NUM INTEGER(5),
COOK_NUM INTEGER(5),
SCORE_NUM INTEGER(5),
PLAY_STATUS VARCHAR(4) NOT NULL,
COURCE_NAME VARCHAR(10) ,
HOLE_DATE DATE(10),
FOREIGN KEY (FOOD_SEAS_NUM) REFERENCES FOOD_SEAS(FOOD_SEAS_NUM)
FOREIGN[*] KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM)
FOREIGN KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
PRIMARY KEY(ROUND_NUM , HOLE_NUM))"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE rounds
(
round_num INTEGER(5) NOT NULL AUTO_INCREMENT,
hole_num INTEGER(1) NOT NULL,
food_seas_num INTEGER(5),
seas_num INTEGER(5),
cook_num INTEGER(5),
score_num INTEGER(5),
play_status VARCHAR(4) NOT NULL,
cource_name VARCHAR(10) ,
hole_date DATE(10),
FOREIGN KEY (food_seas_num) REFERENCES food_seas(food_seas_num)
FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
PRIMARY KEY(round_num , hole_num)) [42001-200]
2024-06-18 15:33:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "SCORES_NUM" が見つかりません
Column "SCORES_NUM" not found; SQL statement:
CREATE TABLE rounds
(
round_num INTEGER(5) NOT NULL AUTO_INCREMENT,
hole_num INTEGER(1) NOT NULL,
food_seas_num INTEGER(5),
seas_num INTEGER(5),
cook_num INTEGER(5),
score_num INTEGER(5),
play_status VARCHAR(4) NOT NULL,
cource_name VARCHAR(10) ,
hole_date DATE(10),
FOREIGN KEY (food_seas_num) REFERENCES food_seas(food_seas_num),
FOREIGN KEY (seas_num) REFERENCES seas(seas_num),
FOREIGN KEY (seas_num) REFERENCES seas(seas_num),
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num),
FOREIGN KEY (scores_num) REFERENCES scores(scores_num),
PRIMARY KEY(round_num , hole_num)) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:216)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 15:37:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE TOURS
(
TOUR_NUM INTEGER(5) NOT NULL AUTO_INCREMENT PRIMARY KEY,
USER_NUM INTEGER(5) NOT NULL,
TOUR_STA DATE(10) NOT NULL,
TOUR_FIN DATE(10),
ROUND_ST INTEGER(5)NOT NULL ,
ROUND_ND INTEGER(5),
ROUND_TH INTEGER(5),
ROUND_STATUS CHAR(2)NOT NULL
FOREIGN[*] KEY (ROUND_ST) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (ROUND_ND) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (ROUND_TH) REFERENCES ROUNDS(ROUND_NUM)
)"; 期待されるステートメント "AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE TOURS
(
TOUR_NUM INTEGER(5) NOT NULL AUTO_INCREMENT PRIMARY KEY,
USER_NUM INTEGER(5) NOT NULL,
TOUR_STA DATE(10) NOT NULL,
TOUR_FIN DATE(10),
ROUND_ST INTEGER(5)NOT NULL ,
ROUND_ND INTEGER(5),
ROUND_TH INTEGER(5),
ROUND_STATUS CHAR(2)NOT NULL
FOREIGN[*] KEY (ROUND_ST) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (ROUND_ND) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (ROUND_TH) REFERENCES ROUNDS(ROUND_NUM)
)"; expected "AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE tours
(
tour_num INTEGER(5) NOT NULL AUTO_INCREMENT PRIMARY KEY,
user_num INTEGER(5) NOT NULL,
tour_sta DATE(10) NOT NULL,
tour_fin DATE(10),
round_st INTEGER(5)NOT NULL ,
round_nd INTEGER(5),
round_th INTEGER(5),
round_status CHAR(2)NOT NULL
FOREIGN KEY (round_st) REFERENCES rounds(round_num)
FOREIGN KEY (round_nd) REFERENCES rounds(round_num)
FOREIGN KEY (round_th) REFERENCES rounds(round_num)
) [42001-200]
2024-06-18 15:38:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE TOURS
(
TOUR_NUM INTEGER(5) NOT NULL AUTO_INCREMENT PRIMARY KEY,
USER_NUM INTEGER(5) NOT NULL,
TOUR_STA DATE(10) NOT NULL,
TOUR_FIN DATE(10),
ROUND_ST INTEGER(5)NOT NULL ,
ROUND_ND INTEGER(5),
ROUND_TH INTEGER(5),
ROUND_STATUS CHAR(2)NOT NULL,
FOREIGN KEY (ROUND_ST) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN[*] KEY (ROUND_ND) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (ROUND_TH) REFERENCES ROUNDS(ROUND_NUM)
)"; 期待されるステートメント "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"
Syntax error in SQL statement "CREATE TABLE TOURS
(
TOUR_NUM INTEGER(5) NOT NULL AUTO_INCREMENT PRIMARY KEY,
USER_NUM INTEGER(5) NOT NULL,
TOUR_STA DATE(10) NOT NULL,
TOUR_FIN DATE(10),
ROUND_ST INTEGER(5)NOT NULL ,
ROUND_ND INTEGER(5),
ROUND_TH INTEGER(5),
ROUND_STATUS CHAR(2)NOT NULL,
FOREIGN KEY (ROUND_ST) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN[*] KEY (ROUND_ND) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (ROUND_TH) REFERENCES ROUNDS(ROUND_NUM)
)"; expected "INDEX, ON, NOT, DEFERRABLE, NOCHECK, CHECK, ,, )"; SQL statement:
CREATE TABLE tours
(
tour_num INTEGER(5) NOT NULL AUTO_INCREMENT PRIMARY KEY,
user_num INTEGER(5) NOT NULL,
tour_sta DATE(10) NOT NULL,
tour_fin DATE(10),
round_st INTEGER(5)NOT NULL ,
round_nd INTEGER(5),
round_th INTEGER(5),
round_status CHAR(2)NOT NULL,
FOREIGN KEY (round_st) REFERENCES rounds(round_num)
FOREIGN KEY (round_nd) REFERENCES rounds(round_num)
FOREIGN KEY (round_th) REFERENCES rounds(round_num)
) [42001-200]
2024-06-18 15:38:51 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOURS_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOURS_NUM,ROUNDS_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOURS_NUM) REFERENCES TOURS(TOURS_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; 期待されるステートメント "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOURS_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOURS_NUM,ROUNDS_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOURS_NUM) REFERENCES TOURS(TOURS_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tours_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tours_num,rounds_num,hole_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
FOREIGN KEY (user_num) REFERENCES users(user_num)
FOREIGN KEY (tours_num) REFERENCES tours(tours_num)
FOREIGN KEY (round_num) REFERENCES rounds(round_num)
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num)
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta)
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42001-200]
2024-06-18 15:39:06 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "ROUNDS_NUM" が見つかりません
Column "ROUNDS_NUM" not found; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tours_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tours_num,rounds_num,hole_num),
FOREIGN KEY (scores_num) REFERENCES scores(scores_num),
FOREIGN KEY (user_num) REFERENCES users(user_num),
FOREIGN KEY (tours_num) REFERENCES tours(tours_num),
FOREIGN KEY (round_num) REFERENCES rounds(round_num),
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num),
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta),
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin),
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:123)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 15:39:32 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "TOURS_NUM" が見つかりません
Column "TOURS_NUM" not found; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tours_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tours_num,round_num,hole_num),
FOREIGN KEY (scores_num) REFERENCES scores(scores_num),
FOREIGN KEY (user_num) REFERENCES users(user_num),
FOREIGN KEY (tours_num) REFERENCES tours(tours_num),
FOREIGN KEY (round_num) REFERENCES rounds(round_num),
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num),
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta),
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin),
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:231)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
=======
	... 21 more
<<<<<<< HEAD
=======
2024-06-18 13:42:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "INSERT INTO SEAS DROP[*] COLUMN SEAS_NUM INTEGER"; 期待されるステートメント "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, WITH, SELECT, TABLE, VALUES"
Syntax error in SQL statement "INSERT INTO SEAS DROP[*] COLUMN SEAS_NUM INTEGER"; expected "., (, DIRECT, SORTED, DEFAULT, VALUES, SET, (, WITH, SELECT, TABLE, VALUES"; SQL statement:
INSERT INTO seas DROP COLUMN seas_num INTEGER [42001-200]
2024-06-18 13:44:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "テーブル名" が見つかりません
Table "テーブル名" not found; SQL statement:
ALTER TABLE テーブル名 ADD COLUMN user_num INTEGER [42102-200]
2024-06-18 13:45:40 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: 不明なデータ型: "BOOREAN"
Unknown data type: "BOOREAN"; SQL statement:
ALTER TABLE seas ADD COLUMN suger BOOREAN [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 13:47:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "SEAS_NUM" にはnull値を許すべきてはありません
Column "SEAS_NUM" must not be nullable; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY (seas_num) [90023-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.mvstore.db.MVTable.addIndex(MVTable.java:334)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:151)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:06:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: 不明なデータ型: "BOOLEAM"
Unknown data type: "BOOLEAM"; SQL statement:
ALTER TABLE seas ADD COLUMN touban BOOLEAM [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:06:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "TENMEN" が重複しています
Duplicate column name "TENMEN"; SQL statement:
ALTER TABLE seas ADD COLUMN tenmen BOOLEAN [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:08:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: 不明なデータ型: 
Unknown data type: ; SQL statement:
ALTER TABLE seas ADD COLUMN herbBOOLEAN [50004-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:505)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.Parser.parseColumnWithType(Parser.java:5971)
	at org.h2.command.Parser.parseColumnForTable(Parser.java:5697)
	at org.h2.command.Parser.parseTableColumnDefinition(Parser.java:8442)
	at org.h2.command.Parser.parseAlterTableAddColumn(Parser.java:8133)
	at org.h2.command.Parser.parseAlterTable(Parser.java:7690)
	at org.h2.command.Parser.parseAlter(Parser.java:6983)
	at org.h2.command.Parser.parsePrepared(Parser.java:887)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:29:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "SEAS" はすでに存在します
Table "SEAS" already exists; SQL statement:
CREATE TABLE seas [42101-200]
2024-06-18 14:29:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "SEAS_NUM" が重複しています
Duplicate column name "SEAS_NUM"; SQL statement:
ALTER TABLE seas ADD COLUMN seas_num INTEGER [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:29:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "USER_NUM" が重複しています
Duplicate column name "USER_NUM"; SQL statement:
ALTER TABLE seas ADD COLUMN user_num INTEGER [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:29:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(seas_num) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:129)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:29:26 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(user_num) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:129)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:29:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "SEAS_NUM" にはnull値を許すべきてはありません
Column "SEAS_NUM" must not be nullable; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(seas_num) [90023-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.mvstore.db.MVTable.addIndex(MVTable.java:334)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:151)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:29:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "USER_NUM" にはnull値を許すべきてはありません
Column "USER_NUM" must not be nullable; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(user_num) [90023-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.mvstore.db.MVTable.addIndex(MVTable.java:334)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:151)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:09 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "SEAS" はすでに存在します
Table "SEAS" already exists; SQL statement:
CREATE TABLE seas [42101-200]
2024-06-18 14:31:09 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "SEAS_NUM" が重複しています
Duplicate column name "SEAS_NUM"; SQL statement:
ALTER TABLE seas ADD COLUMN seas_num INTEGER [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:09 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "USER_NUM" が重複しています
Duplicate column name "USER_NUM"; SQL statement:
ALTER TABLE seas ADD COLUMN user_num INTEGER [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:09 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(user_num) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:129)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "SEAS" はすでに存在します
Table "SEAS" already exists; SQL statement:
CREATE TABLE seas [42101-200]
2024-06-18 14:31:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "SEAS_NUM" が重複しています
Duplicate column name "SEAS_NUM"; SQL statement:
ALTER TABLE seas ADD COLUMN seas_num INTEGER [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列名 "USER_NUM" が重複しています
Duplicate column name "USER_NUM"; SQL statement:
ALTER TABLE seas ADD COLUMN user_num INTEGER [42121-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.setColumns(Table.java:446)
	at org.h2.table.TableBase.<init>(TableBase.java:75)
	at org.h2.table.RegularTable.<init>(RegularTable.java:126)
	at org.h2.mvstore.db.MVTable.<init>(MVTable.java:113)
	at org.h2.mvstore.db.MVTableEngine$Store.createTable(MVTableEngine.java:253)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:127)
	at org.h2.mvstore.db.MVTableEngine.createTable(MVTableEngine.java:41)
	at org.h2.schema.Schema.createTable(Schema.java:697)
	at org.h2.command.ddl.AlterTableAlterColumn.cloneTableStructure(AlterTableAlterColumn.java:414)
	at org.h2.command.ddl.AlterTableAlterColumn.copyData(AlterTableAlterColumn.java:301)
	at org.h2.command.ddl.AlterTableAlterColumn.update(AlterTableAlterColumn.java:210)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(seas_num) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:129)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-18 14:31:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 複数のプライマリキーを定義しようとしました
Attempt to define a second primary key; SQL statement:
ALTER TABLE seas ADD PRIMARY KEY(user_num) [90017-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.message.DbException.get(DbException.java:170)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:129)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
>>>>>>> 2ecd0083c734e57d6f5cea4ddf5683e0be5a28d2
<<<<<<< HEAD
>>>>>>> f6c9659f3d18b339c0717751cf3926bca605c108
2024-06-19 08:57:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 制約 "ROUND_NUM" が見つかりません
Constraint "ROUND_NUM" not found; SQL statement:
ALTER TABLE tours DROP CONSTRAINT round_num [90057-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:41)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 08:57:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 制約 "ROUND_ST" が見つかりません
Constraint "ROUND_ST" not found; SQL statement:
ALTER TABLE tours DROP CONSTRAINT round_st [90057-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:41)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 08:58:09 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 制約 "ROUND_NUM" が見つかりません
Constraint "ROUND_NUM" not found; SQL statement:
ALTER TABLE tours DROP CONSTRAINT round_num [90057-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.command.ddl.AlterTableDropConstraint.update(AlterTableDropConstraint.java:41)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 08:58:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" が依存しているため、"TOURS" をドロップすることはできません
Cannot drop "TOURS" because "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" depends on it; SQL statement:
DROP TABLE tours [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 08:58:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" が依存しているため、"TOURS" をドロップすることはできません
Cannot drop "TOURS" because "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" depends on it; SQL statement:
DROP TABLE tours [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 08:59:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "1" を解析できません
Cannot parse "DATE" constant "1"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: 1
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 20 more
2024-06-19 08:59:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "1" を解析できません
Cannot parse "DATE" constant "1"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: 1
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 20 more
2024-06-19 09:00:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"
Data conversion error converting "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22018-200]
2024-06-19 09:00:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"
Data conversion error converting "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22018-200]
2024-06-19 09:00:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数  を解析できません
Cannot parse "DATE" constant ; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.base/java.lang.StringLatin1.charAt(StringLatin1.java:47)
	at java.base/java.lang.String.charAt(String.java:693)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:145)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 20 more
2024-06-19 09:00:21 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数  を解析できません
Cannot parse "DATE" constant ; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 0
	at java.base/java.lang.StringLatin1.charAt(StringLatin1.java:47)
	at java.base/java.lang.String.charAt(String.java:693)
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:145)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 20 more
2024-06-19 09:00:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"
Data conversion error converting "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22018-200]
2024-06-19 09:00:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"
Data conversion error converting "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22018-200]
2024-06-19 09:00:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"
Data conversion error converting "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22018-200]
2024-06-19 09:00:45 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: データ変換中にエラーが発生しました "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"
Data conversion error converting "'' (TOURS: ""USER_NUM"" INTEGER NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22018-200]
2024-06-19 09:03:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" が依存しているため、"TOURS" をドロップすることはできません
Cannot drop "TOURS" because "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" depends on it; SQL statement:
DROP TABLE tours [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 09:03:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" が依存しているため、"TOURS" をドロップすることはできません
Cannot drop "TOURS" because "CONSTRAINT_E917, CONSTRAINT_E917D0E, CONSTRAINT_E917D0EE_0" depends on it; SQL statement:
DROP TABLE tours [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 09:06:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:06:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:11:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_ST"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_ST"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_st [42001-200]
2024-06-19 09:12:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:13:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:13:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:13:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:13:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:13:05 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:13:15 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW TABLE[*] TOURS"
Syntax error in SQL statement "SHOW TABLE[*] TOURS"; SQL statement:
SHOW TABLE tours [42000-200]
2024-06-19 09:13:33 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:13:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:14:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:14:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:16:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "REFERENCED_TABLE_NAME" が見つかりません
Column "REFERENCED_TABLE_NAME" not found; SQL statement:
SELECT
    CONSTRAINT_NAME,
    TABLE_NAME,
    COLUMN_NAME,
    REFERENCED_TABLE_NAME,
    REFERENCED_COLUMN_NAME
FROM
    INFORMATION_SCHEMA.KEY_COLUMN_USAGE
WHERE
    TABLE_SCHEMA = 'your_database_name'  -- あなたのデータベース名
    AND TABLE_NAME = 'tours'
    AND CONSTRAINT_NAME LIKE '%round_num%' [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 09:19:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY `ROUND_NUM`"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY `ROUND_NUM`"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY `round_num` [42001-200]
2024-06-19 09:19:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY `ROUND_ST`"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY `ROUND_ST`"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY `round_st` [42001-200]
2024-06-19 09:20:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "USER_CONSTRAINTS" が見つかりません
Table "USER_CONSTRAINTS" not found; SQL statement:
SELECT * FROM user_constraints [42102-200]
2024-06-19 09:22:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "NULL" を解析できません
Cannot parse "DATE" constant "NULL"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: NULL
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 20 more
2024-06-19 09:22:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: "DATE" 定数 "NULL" を解析できません
Cannot parse "DATE" constant "NULL"; SQL statement:
INSERT INTO "PUBLIC"."TOURS"("TOUR_NUM","USER_NUM","TOUR_STA","TOUR_FIN","ROUND_ST","ROUND_ND","ROUND_TH","ROUND_STATUS")VALUES(?,?,?,?,?,?,?,?) [22007-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:457)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:194)
	at org.h2.value.ValueDate.parse(ValueDate.java:74)
	at org.h2.value.Value.convertToDate(Value.java:1093)
	at org.h2.value.Value.convertTo(Value.java:816)
	at org.h2.value.Value.convertTo(Value.java:772)
	at org.h2.value.TypeInfo.cast(TypeInfo.java:515)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:378)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:845)
	at org.h2.command.dml.Insert.insertRows(Insert.java:187)
	at org.h2.command.dml.Insert.update(Insert.java:151)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdateInternal(JdbcPreparedStatement.java:191)
	at org.h2.jdbc.JdbcPreparedStatement.executeUpdate(JdbcPreparedStatement.java:152)
	at org.h2.result.UpdatableRow.insertRow(UpdatableRow.java:324)
	at org.h2.jdbc.JdbcResultSet.insertRow(JdbcResultSet.java:3015)
	at org.h2.server.web.WebApp.editResult(WebApp.java:1138)
	at org.h2.server.web.WebApp.process(WebApp.java:247)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:173)
	at org.h2.server.web.WebThread.process(WebThread.java:139)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.IllegalArgumentException: NULL
	at org.h2.util.DateTimeUtils.parseDateValue(DateTimeUtils.java:166)
	at org.h2.value.ValueDate.parse(ValueDate.java:72)
	... 20 more
2024-06-19 09:23:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: 列 "ROUND_STATUS CHAR(2)" の値が長過ぎます: "STRINGDECODE('\u8abf\u7406\u524d') (3)"
Value too long for column "ROUND_STATUS CHAR(2)": "STRINGDECODE('\u8abf\u7406\u524d') (3)"; SQL statement:
INSERT INTO tours
VALUES(1,1,'2024-06-18',null,1,1,1,'調理前') [22001-200]
2024-06-19 09:27:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:28:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_ST"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_ST"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY round_st [42001-200]
2024-06-19 09:28:14 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_ND"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_ND"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY round_nd [42001-200]
2024-06-19 09:28:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_TH"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_TH"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY round_th [42001-200]
2024-06-19 09:30:23 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SELECT FOREIGN[*] KEY
FROM INFORMATION_SCHEMA.FOREIGN KEY
WHERE TABLE_NAME = 'tours' 
  AND FOREIGN KEY_TYPE = 'REFERENTIAL'"; 期待されるステートメント "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS, UNIQUE"
Syntax error in SQL statement "SELECT FOREIGN[*] KEY
FROM INFORMATION_SCHEMA.FOREIGN KEY
WHERE TABLE_NAME = 'tours' 
  AND FOREIGN KEY_TYPE = 'REFERENTIAL'"; expected "TOP, LIMIT, DISTINCT, ALL, *, NOT, EXISTS, INTERSECTS, UNIQUE"; SQL statement:
SELECT FOREIGN KEY
FROM INFORMATION_SCHEMA.FOREIGN KEY
WHERE TABLE_NAME = 'tours' 
  AND FOREIGN KEY_TYPE = 'REFERENTIAL' [42001-200]
2024-06-19 09:32:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SHOW CREATE[*] TABLE TOURS"
Syntax error in SQL statement "SHOW CREATE[*] TABLE TOURS"; SQL statement:
SHOW CREATE TABLE tours [42000-200]
2024-06-19 09:32:34 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY FK_ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY FK_ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY fk_round_num [42001-200]
2024-06-19 09:32:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:32:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY (ROUND_NUM)"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY (ROUND_NUM)"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY (round_num) [42001-200]
2024-06-19 09:33:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS
DROP FOREIGN[*] KEY 'round_num'"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS
DROP FOREIGN[*] KEY 'round_num'"; expected "identifier"; SQL statement:
ALTER TABLE tours
DROP FOREIGN KEY 'round_num' [42001-200]
2024-06-19 09:35:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE ROUNDS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE ROUNDS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE rounds DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:35:42 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE ROUNDS DROP FOREIGN[*] KEY ROUND_ST"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE ROUNDS DROP FOREIGN[*] KEY ROUND_ST"; expected "identifier"; SQL statement:
ALTER TABLE rounds DROP FOREIGN KEY round_st [42001-200]
2024-06-19 09:36:48 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:37:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:37:43 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:37:53 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_ST"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_ST"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_st [42001-200]
2024-06-19 09:38:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:38:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; 期待されるステートメント "identifier"
Syntax error in SQL statement "ALTER TABLE TOURS DROP FOREIGN[*] KEY ROUND_NUM"; expected "identifier"; SQL statement:
ALTER TABLE tours DROP FOREIGN KEY round_num [42001-200]
2024-06-19 09:48:52 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOURS_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOURS_NUM,ROUNDS_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOURS_NUM) REFERENCES TOURS(TOURS_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; 期待されるステートメント "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOURS_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOURS_NUM,ROUNDS_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOURS_NUM) REFERENCES TOURS(TOURS_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tours_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tours_num,rounds_num,hole_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
FOREIGN KEY (user_num) REFERENCES users(user_num)
FOREIGN KEY (tours_num) REFERENCES tours(tours_num)
FOREIGN KEY (round_num) REFERENCES rounds(round_num)
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num)
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta)
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42001-200]
2024-06-19 09:49:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOUR_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOUR_NUM,ROUND_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOUR_NUM) REFERENCES TOURS(TOUR_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; 期待されるステートメント "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOUR_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOUR_NUM,ROUND_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOUR_NUM) REFERENCES TOURS(TOUR_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tour_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tour_num,round_num,hole_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
FOREIGN KEY (user_num) REFERENCES users(user_num)
FOREIGN KEY (tour_num) REFERENCES tours(tour_num)
FOREIGN KEY (round_num) REFERENCES rounds(round_num)
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num)
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta)
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42001-200]
2024-06-19 09:52:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOUR_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOUR_NUM,ROUND_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOUR_NUM) REFERENCES TOURS(TOUR_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; 期待されるステートメント "INDEX, ,, )"
Syntax error in SQL statement "CREATE TABLE PAST_SCORES(
SCORES_NUM INTEGER(5) NOT NULL,
USER_NUM INTEGER(5) NOT NULL,
TOUR_NUM INTEGER(5) NOT NULL,
ROUND_NUM INTEGER(5) NOT NULL,
HOLE_NUM INTEGER(2) NOT NULL,
TOUR_STA DATE NOT NULL,
TOUR_FIN DATE NOT NULL,
COOK_NUM INTEGER(5) NOT NULL,
SCORES INTEGER(3) NOT NULL,
PRIMARY KEY(TOUR_NUM,ROUND_NUM,HOLE_NUM)
FOREIGN[*] KEY (SCORES_NUM) REFERENCES SCORES(SCORES_NUM)
FOREIGN KEY (USER_NUM) REFERENCES USERS(USER_NUM)
FOREIGN KEY (TOUR_NUM) REFERENCES TOURS(TOUR_NUM)
FOREIGN KEY (ROUND_NUM) REFERENCES ROUNDS(ROUND_NUM)
FOREIGN KEY (HOLE_NUM) REFERENCES ROUNDS(HOLE_NUM)
FOREIGN KEY (TOUR_STA) REFERENCES TOURS(TOUR_STA)
FOREIGN KEY (TOUR_FIN) REFERENCES TOURS(TOUR_FIN)
FOREIGN KEY (COOK_NUM) REFERENCES COOKS(COOK_NUM))"; expected "INDEX, ,, )"; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tour_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE NOT NULL,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tour_num,round_num,hole_num)
FOREIGN KEY (scores_num) REFERENCES scores(scores_num)
FOREIGN KEY (user_num) REFERENCES users(user_num)
FOREIGN KEY (tour_num) REFERENCES tours(tour_num)
FOREIGN KEY (round_num) REFERENCES rounds(round_num)
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num)
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta)
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin)
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42001-200]
2024-06-19 09:55:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "PAST_SCORES" はすでに存在します
Table "PAST_SCORES" already exists; SQL statement:
CREATE TABLE past_scores(
scores_num INTEGER(5) NOT NULL,
user_num INTEGER(5) NOT NULL,
tour_num INTEGER(5) NOT NULL,
round_num INTEGER(5) NOT NULL,
hole_num INTEGER(2) NOT NULL,
tour_sta DATE NOT NULL,
tour_fin DATE,
cook_num INTEGER(5) NOT NULL,
scores INTEGER(3) NOT NULL,
PRIMARY KEY(tour_num,round_num,hole_num),
FOREIGN KEY (scores_num) REFERENCES scores(scores_num),
FOREIGN KEY (user_num) REFERENCES users(user_num),
FOREIGN KEY (tour_num) REFERENCES tours(tour_num),
FOREIGN KEY (round_num) REFERENCES rounds(round_num),
FOREIGN KEY (hole_num) REFERENCES rounds(hole_num),
FOREIGN KEY (tour_sta) REFERENCES tours(tour_sta),
FOREIGN KEY (tour_fin) REFERENCES tours(tour_fin),
FOREIGN KEY (cook_num) REFERENCES cooks(cook_num)) [42101-200]
2024-06-19 09:55:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "更新数[*]: 0"
Syntax error in SQL statement "更新数[*]: 0"; SQL statement:
更新数: 0 [42000-200]
2024-06-19 09:56:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "更新数[*]: 0"
Syntax error in SQL statement "更新数[*]: 0"; SQL statement:
更新数: 0 [42000-200]
2024-06-19 10:18:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: テーブル "COOK_SEAS" が見つかりません
Table "COOK_SEAS" not found; SQL statement:
INSERT INTO cook_seas
VALUES(2,2,true,'なし') [42102-200]
2024-06-19 10:22:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "SEAS_NUM" が見つかりません
Column "SEAS_NUM" not found; SQL statement:
INSERT INTO food_seas(seas_num,user_num)
VALUES(2,1) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.command.Parser.parseColumn(Parser.java:1450)
	at org.h2.command.Parser.parseColumnList(Parser.java:1435)
	at org.h2.command.Parser.parseInsertGivenTable(Parser.java:1804)
	at org.h2.command.Parser.parseInsert(Parser.java:1749)
	at org.h2.command.Parser.parsePrepared(Parser.java:954)
	at org.h2.command.Parser.parse(Parser.java:843)
	at org.h2.command.Parser.parse(Parser.java:815)
	at org.h2.command.Parser.prepareCommand(Parser.java:738)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 10:29:44 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "CREATE TABLE COOKS_SEAS(
    COOKS_SEAS_NUM  INT(5)          NOT NULL,
    SOLTY           BOOLEAN         ,
    REGION          CHAR(10)
    FOREIGN[*] KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
)"; 期待されるステートメント "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"
Syntax error in SQL statement "CREATE TABLE COOKS_SEAS(
    COOKS_SEAS_NUM  INT(5)          NOT NULL,
    SOLTY           BOOLEAN         ,
    REGION          CHAR(10)
    FOREIGN[*] KEY (SEAS_NUM) REFERENCES SEAS(SEAS_NUM)
)"; expected "FOR, UNSIGNED, INVISIBLE, VISIBLE, NOT, NULL, AS, DEFAULT, GENERATED, ON, NOT, NULL, AUTO_INCREMENT, BIGSERIAL, SERIAL, IDENTITY, NULL_TO_DEFAULT, SEQUENCE, SELECTIVITY, COMMENT, CONSTRAINT, PRIMARY, UNIQUE, NOT, NULL, COMMENT, CHECK, REFERENCES, ,, )"; SQL statement:
CREATE TABLE cooks_seas(
    cooks_seas_num  INT(5)          not null,
    solty           BOOLEAN         ,
    region          CHAR(10)
    FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
) [42001-200]
2024-06-19 10:29:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "SEAS_NUM" が見つかりません
Column "SEAS_NUM" not found; SQL statement:
CREATE TABLE cooks_seas(
    cooks_seas_num  INT(5)          not null,
    solty           BOOLEAN         ,
    region          CHAR(10),
    FOREIGN KEY (seas_num) REFERENCES seas(seas_num)
) [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.table.Table.getColumn(Table.java:703)
	at org.h2.table.IndexColumn.mapColumns(IndexColumn.java:115)
	at org.h2.command.ddl.AlterTableAddConstraint.tryUpdate(AlterTableAddConstraint.java:216)
	at org.h2.command.ddl.AlterTableAddConstraint.update(AlterTableAddConstraint.java:78)
	at org.h2.command.ddl.CommandWithColumns.createConstraints(CommandWithColumns.java:83)
	at org.h2.command.ddl.CreateTable.update(CreateTable.java:133)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 10:42:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*] SEAS_NUM VARCHAR(5) NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*] SEAS_NUM VARCHAR(5) NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY seas_num VARCHAR(5) NOT NULL [42001-200]
2024-06-19 10:44:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*] SEAS_NUM VARCHAR NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*] SEAS_NUM VARCHAR NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY seas_num VARCHAR NOT NULL [42001-200]
2024-06-19 10:45:10 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*] COLUMN SEAS_NUM VARCHAR NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*] COLUMN SEAS_NUM VARCHAR NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY COLUMN seas_num VARCHAR NOT NULL [42001-200]
2024-06-19 10:46:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY  seas_num VARCHAR(5) NOT NULL [42001-200]
2024-06-19 10:46:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY  seas_num VARCHAR(5) NOT NULL [42001-200]
2024-06-19 10:46:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY  seas_num VARCHAR(5) NOT NULL [42001-200]
2024-06-19 10:46:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY  seas_num VARCHAR(5) NOT NULL [42001-200]
2024-06-19 10:46:55 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; 期待されるステートメント "., ADD, SET, RENAME, DROP, ALTER"
Syntax error in SQL statement "ALTER TABLE SEAS
MODIFY[*]  SEAS_NUM VARCHAR(5) NOT NULL"; expected "., ADD, SET, RENAME, DROP, ALTER"; SQL statement:
ALTER TABLE seas
MODIFY  seas_num VARCHAR(5) NOT NULL [42001-200]
2024-06-19 10:49:37 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: "CONSTRAINT_906, CONSTRAINT_906E" が依存しているため、"SEAS" をドロップすることはできません
Cannot drop "SEAS" because "CONSTRAINT_906, CONSTRAINT_906E" depends on it; SQL statement:
DROP TABLE seas [90107-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:576)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.command.ddl.DropTable.prepareDrop(DropTable.java:98)
	at org.h2.command.ddl.DropTable.update(DropTable.java:124)
	at org.h2.command.CommandContainer.update(CommandContainer.java:198)
	at org.h2.command.Command.executeUpdate(Command.java:251)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:228)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
=======
=======
>>>>>>> e05a3a060b0db45c394a8a2d621319bb5246bc59
>>>>>>> ba7279739aeb88f2699e7368b048b11a35158cfc
>>>>>>> 38a5ea1de012b36ba58b69404b577a8e96895230
2024-06-19 13:38:26 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "COOK" が見つかりません
Column "COOK" not found; SQL statement:
SELECT * FROM COOKS where cook=num =1 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.expression.condition.Comparison.optimize(Comparison.java:190)
	at org.h2.command.dml.Select.prepare(Select.java:1209)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 13:40:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "ROUND" が見つかりません
Column "ROUND" not found; SQL statement:
SELECT round_st ,round_nd,round,round_th FROM TOURS WHERE USER_NUM =1 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 13:41:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: 列 "ROUND" が見つかりません
Column "ROUND" not found; SQL statement:
SELECT round_st ,round_nd,round,round_th FROM TOURS WHERE USER_NUM =1 [42122-200]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:453)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:429)
	at org.h2.message.DbException.get(DbException.java:205)
	at org.h2.message.DbException.get(DbException.java:181)
	at org.h2.expression.ExpressionColumn.getColumnException(ExpressionColumn.java:163)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:145)
	at org.h2.command.dml.Select.prepare(Select.java:1206)
	at org.h2.command.Parser.prepareCommand(Parser.java:744)
	at org.h2.engine.Session.prepareLocal(Session.java:657)
	at org.h2.engine.Session.prepareCommand(Session.java:595)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1235)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:212)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:201)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1459)
	at org.h2.server.web.WebApp.query(WebApp.java:1116)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1078)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1065)
	at org.h2.server.web.WebThread.process(WebThread.java:178)
	at org.h2.server.web.WebThread.run(WebThread.java:94)
	at java.base/java.lang.Thread.run(Thread.java:834)
2024-06-19 13:43:22 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: SQLステートメントに文法エラーがあります "SELECT * FROM COOKS WHERE COOK_NUM -=[*]1"
Syntax error in SQL statement "SELECT * FROM COOKS WHERE COOK_NUM -=[*]1"; SQL statement:
SELECT * FROM COOKS WHERE COOK_NUM -=1 [42000-200]
